dyn_em/solve_em.F

```diff
@@ -128,6 +128,8 @@ SUBROUTINE solve_em ( grid , config_flags  &
    ! Flag for producing diagnostic fields (e.g., radar reflectivity)
    LOGICAL                        :: diag_flag
    INTEGER                        :: ke_diag ! tells reflectivity calculation whether to do full depth or only k=1
+   LOGICAL                        :: restart_flag
+   LOGICAL                        :: restart_run
```

Comment:
@negin513 Is restart_run used anywhere?

Reply:
Thanks @weiwangncar . Correct! `restart_run` variable was previously used here but  now it is not being used here at all.

Reply:
I remove the unused variable.